pull_capture_data.py

```diff
@@ -15,9 +41,16 @@
 output1=output1.split()
 #print output1[1]
 
+
+## Python is dynamic you do not need to 
+## define them to used them just init 
+## them before
+
 directory = "LOL"
 folder = "OLO"
 
+## here i change to model and product since i have them already in diferent 
+## variables 
```

Comment:
This is not the updated code. Please check the new file with your changes added.